server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java

```diff
@@ -0,0 +1,29 @@
+package org.mixit.cesar.site.utils;
+
+import static org.assertj.core.api.StrictAssertions.assertThat;
+import static org.junit.Assert.*;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+/**
+ * @author Dev-Mind <guillaume@dev-mind.fr>
+ * @since 22/02/16.
+ */
+public class CamelCaseTest {
+
+    @Test
+    public void should_return_null_when_string_null(){
+        assertThat(CamelCase.camelCase(null)).isNull();
+    }
+
+    @Test
+    public void should_return_A_when_string_is_a(){
+        assertThat(CamelCase.camelCase("a")).isEqualTo("A");
+    }
+
+    @Test
+    public void should_convertString(){
+        assertThat(CamelCase.camelCase("Guillaume")).isEqualTo("guillaume");
```

Comment:
Yes if we use a common lib, I'll do that but I'm not a fan to import a long library for a very simple needing. 
In Java 8 I think we could also write a simpler code with lastname.chars()